build/x86_64_sles15/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM opensuse/leap:15.1
+
+RUN zypper -n refresh \
+ && zypper -n install \
+        autoconf \
+        bzip2 \
+        curl \
+        expect \
+        gcc-c++ \
+        git \
+        patch \
+        ruby \
+        rpm-build \
+        which \
+        zlib-devel \
+  # RVM GPG keys
+  && gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
+  && (curl -sSL https://get.rvm.io | /bin/bash -s stable) \
+  # RVM requires running in a login shell.
+  && /bin/bash -l -c "rvm requirements && rvm install 2.5.3 && gem install bundler --no-document && gem update" \
```

Comment:
If we're going to upgrade to 2.5, let's do it across the board.

Reply:
Meaning upgrade all dockerfiles under build folder to 2.5.3?

Reply:
Yup. This can probably be in a separate PR though.